WebHostLib/tracker.py

```diff
@@ -422,11 +422,11 @@ def render_generic_multiworld_tracker(tracker_data: TrackerData, enabled_tracker
 
 if "Factorio" in network_data_package["games"]:
     def render_Factorio_multiworld_tracker(tracker_data: TrackerData, enabled_trackers: List[str]):
-        inventories: Dict[TeamPlayer, Dict[int, int]] = {
-            (team, player): {
+        inventories: Dict[TeamPlayer, collections.Counter[str]] = {
```

Comment:
This `collections.Counter[str]` type annotation won't work in Python 3.8.
```suggestion
        inventories: Dict[TeamPlayer, Counter[str]] = {
```

Another workaround to fix it without `typing.Counter` is just to put the whole type annotation in quotes.